### ISSUE TITLE
absent: ext-button's OnDirectClick property

### ISSUE THREAD

**Issue by fabriciomurta** [extnet/Ext.NET#1700]
The `ext-button` component's `OnDirectClick` shorthand for the `Click` direct event is missing. It is defined in Ext.NET 5.1.0's `Ext.Net.ButtonBase` class.

[objectdotnet/ext7-sandbox:buttons/basic/loadingstate/index.cshtml#L21](https://github.com/objectdotnet/ext7-sandbox/blob/04b9618e95e849678bde58b469bc4466aa450fbf/ExtNetCore7-20200325/Pages/examples/buttons/basic/loadingstate/index.cshtml#L21)
```xml
<ext-button
    Text="Click Me"
    AutoLoadingState="true"
    OnDirectClick="Button1_Click"
    />
```

**Expected client-side output:**
```js
Ext.create("Ext.button.Button", {
    id: "ctl02",
    renderTo: "App.ctl02_Container",
    text: "Click Me",
    loadingState: true,
    directEvents: {
        click: {
            fn: function(item, e) {
                Ext.net.directRequest({
                    control: this
                });
            }
        }
    }
});
```

**Actual client-side output:**
```js
Ext.create("Ext.button.Button", {
    id: "ctl01",
    renderTo: "App.ctl01_Container",
    text: "Click Me"
});
```

Notes:
- the missing `AutoLoadingState` will be handled in #1699

**Comment by AndreyChechel** [comment]
Implemented in `7.0.0-preview4`

**Comment by geoffreymcgill** [comment]
Here's a very basic sample demonstrating `OnDirectClick`. 

Please note, the full URL path to the handler must be specified. In the following example, the `TestModel` requires `OnDirectClick="Text/DoSomething"` instead of `OnDirectClick="DoSomething"` which not resolve correctly. 

```cshtml
@page  "{handler?}"
@using Ext.Net;
@using Ext.Net.Core;

@model demo1.Pages.TestModel
@{
    Layout = null;
}

<!DOCTYPE html>

<html>
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Ext.NET OnDirectClick demo</title>
</head>
<body>

    <ext-button Text="Submit" OnDirectClick="Test/DoSomething" />

</body>
</html>
```

```csharp
using System;
using Microsoft.AspNetCore.Mvc;
using Microsoft.AspNetCore.Mvc.RazorPages;

using Ext.Net;
using Ext.Net.Core;

namespace demo1.Pages
{
    [DirectModel]
    public class TestModel : PageModel
    {
        public void OnGet() { }

        [Direct]
        public IActionResult OnPostDoSomething()
        {
            this.X().Toast("DoSomething");

            return this.Direct();
        }
    }
}
```
```
